src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -656,7 +658,7 @@ namespace cryptonote
       return false;
     }
 
-    auto lns_db_file_path = folder / "lns.db";
+    auto ons_db_file_path = folder / "ons.db";
```

Comment:
(See comment about this in #1428 -- I'm not entirely sure which PR this originates in).